content/en/security/cspm/getting_started.md

```diff
@@ -21,30 +21,186 @@ Cloud Security Posture Management is not currently available in this site.
+
+1. Navigate to **Security** > **Setup & Configuration**.
+2. Follow the [in-app instructions][2] to activate CSPM for your account.
+3. On the **Setup & Configuration** > **Cloud Providers** tab, click the **[Azure][3]** tile.
+4. Enable CSPM for your Azure subscriptions by turning on the **CSPM Enabled** toggle.
+
+#### Azure integration tile
+
+1. On the Azure integration tile, select an Azure app.
+2. Under **Resource Collection**, select the **Enable resource collection for Cloud Security Posture Management** checkbox.
+3. Click **Update Configuration**.
+
+**Note**: To disable CSPM, turn off the **CSPM Enabled** toggle for your Azure subscriptions. Your previous findings and homepage will still be available in-app, but you will not incur additional billing costs.
+
+[1]: https://docs.datadoghq.com/integrations/azure
+[2]: https://app.datadoghq.com/security/configuration
+[3]: https://app.datadoghq.com/security/configuration?sectionId=secureCloudEnvironment&secure-cloud-environment=azure
+
+{{% /tab %}}
+
+{{% tab "GCP" %}}
+
+### Set up the Datadog GCP integration
+
+If you haven't already, set up the [Google Cloud Platform integration][1] and make sure that you have successfully completed the steps for enabling [metric collection][2].
+
+### Enable CSPM for CCP
+
+Use one of the following methods to enable CSPM for your GCP projects:
+
+### Security Setup & Configuration
+
+1. Navigate to **Security** > **Setup & Configuration**.
+2. Follow the [in-app instructions][3] to activate CSPM for your account.
+3. On the **Setup & Configuration** > **Cloud Providers** tab, click the **[GCP][4]** tile.
+4. Enable CSPM for your GCP projects by turning on the **CSPM Enabled** toggle.
+
+### GCP integration tile
+
+1. On the GCP integration tile, select a GCP project.
+2. Under **Enable resource collection for Cloud Security Posture Management**, select the **Resource collection** checkbox.
+3. Click **Update Configuration**.
+
+**Note**: To disable CSPM, turn off the **CSPM Enabled** toggle for your GCP projects. Your previous findings and homepage will still be available in-app, but you will not incur additional billing costs.
+
+[1]: https://docs.datadoghq.com/integrations/google_cloud_platform
+[2]: https://docs.datadoghq.com/integrations/google_cloud_platform/#metric-collection
+[3]: https://app.datadoghq.com/security/configuration
+[4]: https://app.datadoghq.com/security/configuration?sectionId=secureCloudEnvironment&secure-cloud-environment=google-cloud-platform
+
+{{% /tab %}}
+
+{{% tab "Docker" %}}
+
+### Enable CSPM for Docker
+
+1. Navigate to **Security** > **Setup & Configuration**.
+2. Follow the [in-app instructions][1] to activate CSPM for your account.
+3. On the **Setup & Configuration** > **Host and containers** tab, click the **[Docker][2]** tile.
+4. Click **Select API key** to choose the API key you want to use with CSPM.
+5. Copy the command and run it in your Docker environment to enable CSPM.
```

Comment:
```suggestion
5. Copy the automatically generated command and run it in your Docker environment to enable CSPM.
```

Reply:
Might be worth noting here in case they're reading the instructions separately from completing the tasks